tests/cpp/test_resize.cpp

```diff
@@ -4096,64 +4108,85 @@ TEST_F(ResizeTest, PropagateSliceToInputs) {
   auto tv0 = makeConcreteTensor(shape);
   fusion.addInput(tv0);
 
-  auto tv1 = set(tv0);
+  // Dont't use set here as it gets taken by the no-op scheduler
+  auto tv1 = sin(tv0);
```

Comment:
The changes from `set` to `sin` or `cos` are just to avoid the preseg transformation from kicking in.